bin/generate.js

```diff
@@ -304,7 +304,7 @@ const getReplacements = async ({ name, namespace, npm }) => {
   }
 
   const nameReplacements = [
-    { regex: /\[genVersion\]/g, value: pjson.version },
+    { regex: /\[genVersion\]/g, value: pjson.version.split('.').join("") },
```

Comment:
Not sure I understand why we are stripping the `.` characters here.

This results in `wcgen-v363` instead `wcgen-v3.6.3`

Reply:
You cannot have dots in repo tags.